ports/stm32/boards/NUCLEO_F446RE/mpconfigboard.h

```diff
@@ -24,12 +24,12 @@
 #define MICROPY_HW_UART_REPL_BAUD   115200
 
 // I2C busses
-#define MICROPY_HW_I2C1_SCL (pin_B6)        // Arduino D10, pin 17 on CN10
-#define MICROPY_HW_I2C1_SDA (pin_B7)        //              pin 21 on CN7
+#define MICROPY_HW_I2C1_SCL (pin_B8)        // Arduino D15, pin 3 on CN10
+#define MICROPY_HW_I2C1_SDA (pin_B9)        //         D14, pin 5 on CN10
```

Comment:
This change looks good, since PB8/PB9 are the "official" I2C pins in the datasheet for the Nucleo board.

Reply:
Without this change I couldn’t get the board to communicate with my I2C peripherals